datadog-accessors.go

```diff
@@ -10089,6 +10089,161 @@ func (i *IntegrationAWSAccountDeleteRequest) SetRoleName(v string) {
 	i.RoleName = &v
 }
 
+// GetLambdaARN returns the LambdaARN field if non-nil, zero value otherwise.
+func (i *IntegrationAWSLambdaARN) GetLambdaARN() string {
```

Comment:
Do we need this one since we have the one below?

Reply:
No, I don't think so. This was generated by the command `make generate` which I ran following an error message that in the test pipeline.

I will investiguate to try to understand why there two similar functions were generated

Reply:
Yeah, perhaps we strictly need all of these, but this is how the code generation works right now and it's not in the scope of this PR to investigate/change that. I'm going to mark all conversations for `datadog-accessors.go` as resolved.